src/helpers/language.ts

```diff
@@ -78,6 +84,10 @@ export function getCountryCodeFromIso(locale: string) : string | undefined {
     return normalizedLocale.split("-")[1];
 }
 
+function isLocaleSupported(locale: string) {
+	return !!localeToStringsMap[normalizeLocaleString(locale)];
```

Comment:
Minor: The `normalizeLocaleString(...)` call is not really an expensive operation, but given the current usage of this internal function, I think the `locale` will always be normalized already.

Reply:
Yeah, I figured it was safer this way because we can't anticipate how it might be used in the future. If it were more expensive, we could just add a note, but it seemed pretty harmless to just put in the extra safeguard.